docs/en/observability/logs-index-template.asciidoc

```diff
@@ -1,44 +1,47 @@
 that configure index mappings, settings, and aliases.
 
-The default Logs index template can be viewed in {kib}.
+You can view the default `logs` index template in {kib}.
 Navigate to **{stack-manage-app}** → **Index Management** → **Index Templates**, and search for `logs`.
-Select any of the logs index templates to view their relevant component templates.
+Select the `logs` index templates to view relevant component templates.
 
 [discrete]
-[custom-logs-template-edit]
-=== Edit the {es} index template
+[[custom-logs-template-edit]]
+== Edit the `logs` index template
 
-{es} has a built-in index template for the `logs-*-*` index pattern, which is composed of component templates including:
+The default `logs` index template for the `logs-*-*` index pattern is composed of the following component templates:
 
 * `logs@mappings`
 * `logs@settings`
 * `logs@custom`
 * `ecs@mappings`
 
-You can use the `@custom` component template to customize your {es} indices. The `logs@custom` template is not installed by default, but you can create a template named `logs@custom` to override and extend default mappings or settings.
-
-Open {kib} and navigate to **{stack-manage-app}** → **Index Management** → **Component Templates**.
+You can use the `logs@custom` component template to customize your {es} indices. The `logs@custom` component template is not installed by default, but you can create a component template named `logs@custom` to override and extend default mappings or settings. To do this:
 
+. Open {kib} and navigate to **{stack-manage-app}** → **Index Management** → **Component Templates**.
 . Click *Create component template*.
 . Name the component template logs@custom.
 . Add any custom metadata, index settings, or mappings.
 
+Changes to component templates are not applied retroactively to existing indices. For changes to take effect, create a new write index for impacted data streams by triggering a rollover. Do this using the {es} {ref}/indices-rollover-index.html[Rollover API]. For example, to roll over the `logs-generic-default` data stream, run:
+
+[source,console]
+----
+POST /logs-generic-default/_rollover/
+----
+
 [discrete]
 [[custom-logs-template-default-field]]
 === Set the `default_field` using the custom template
 
-The `logs-*-*` index template uses `default_field: [*]` meaning queries without specified fields will search across all fields.
+The `logs` index template uses `default_field: [*]` meaning queries without specified fields will search across all fields.
 You can update the `default_field` to  search in the `message` field instead of all fields using the `logs@custom` component template.
 
-Either create the `logs@custom` template as detailed in the previous section, or edit the existing one to include the following in the *Index settings*:
+Either create the `logs@custom` component template as outlined in the previous section, or add the following code to the *Index settings* of the `logs` index template:
```

Comment:
@ruflin, what do you think about this sentence?

Personally, I wouldn't suggest fiddling with the `logs` index template but only changing the custom component template. 

Thoughts? @gbamparop

Reply:
ok. sorry it wasn't 100% clear. 

I would rephrase as "If you don't already have a logs@custom component template you can create it manually. Once you have it, you can add the following to the settings section"

Reply:
Thanks for pointing it out. I updated it to be clear and add steps for updating the template. I guess because of the way the search works, you can't search for anything using the "@" symbol? or it throws the error:
![image](https://github.com/elastic/observability-docs/assets/114418652/8da8cbe1-244e-431f-8ab7-13030466e059)

Not sure if there's an underlying reason for this, but it does make it pretty difficult to find any custom templates.

Reply:
That sounds like a bug especially as we use it in our names. Has anyone reported this? If not, lets do it.